app/resources/serializable_configuration_record.rb

```diff
@@ -0,0 +1,13 @@
+class SerializableConfigurationRecord < SerializableBase
```

Comment:
Also not needed. This surfaces something though. We should have the validity check being in the meta data. I don't think we have a test for that.